core/src/main/java/org/openedx/core/ui/ComposeCommon.kt

```diff
@@ -731,6 +733,8 @@ fun AutoSizeText(
 
 @Composable
 fun DiscoveryCourseItem(course: Course, windowSize: WindowSize, onClick: (String) -> Unit) {
+    val config = getKoin().get<Config>()
```

Comment:
We should keep our Compose functions without injects.
Better to pass config here as an argument.